src/Components/PhoneNumberInput/validatePhoneNumber.ts

```diff
@@ -0,0 +1,61 @@
+import { debounce } from "lodash"
+import { fetchQuery, graphql } from "react-relay"
+import { createRelaySSREnvironment } from "System/Relay/createRelaySSREnvironment"
+import { validatePhoneNumberQuery } from "__generated__/validatePhoneNumberQuery.graphql"
+
+const relayEnvironment = createRelaySSREnvironment()
```

Comment:
I'm just passing in an unauthenticated Relay environment. This one won't have a cache that's hydrated from the store as well. Doesn't matter.